### PR TITLE
Add update support

### PR DESCRIPTION
**Pull request by kentikethan**
added device update support
changed all code to conform to ansible standards
added documentation

### CODE REVIEW

File: playbooks/tmp/example_device.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-        "device_subtype": "router",
-        "minimize_snmp": false,
-        "plan_name": "Free Flowpak Plan 2",
-        "sending_ips": ["10.1.7.200"],
-        "site_name": "GEG",
-        "device_bgp_type": "none",
+        "deviceName": "access_switch_100",
+        "deviceSampleRate": 10,
+        "deviceSnmpCommunity": "kentik",
+        "deviceSnmpIp": "192.168.1.100",
```
Comment (jryburn) [inline playbooks/tmp/example_device.json]: Probably should use IP addresses in the example that are from the RFC5737 space reserved for that purpose (https://datatracker.ietf.org/doc/rfc5737/).

**Review by jryburn — CHANGES_REQUESTED** [review]
Probably should use IP addresses in the example that are from the RFC5737 space reserved for that purpose (https://datatracker.ietf.org/doc/rfc5737/).

Also, is playbooks/tmp the right place for the examples json files? I think both the playbook examples and json examples could just go in the playbooks directory. Just my 2 cents.

File: playbooks/tmp/example_device.json (hunk 1 above)
Comment (kentikethan) [reply playbooks/tmp/example_device.json]: done

Comment (kentikethan) [reply playbooks/tmp/example_device.json]: also renamed the tmp directory to files to match ansible best practices. does that work?

Comment (kentikethan) [reply playbooks/tmp/example_device.json]: https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html

**Review by jryburn — APPROVED** [review]
LGTM

**Pull request merged**